### PR TITLE
Youtube Clone Project 

### PR DESCRIPTION
**Pull request by KartheeKWD**
-Stickler Enabled
-Validated HTML Markup.
- Styling Errors fixed
- Navigation Bar fixed and made Responsive
-Readme updated with Live Demo  and screenshot 
-Added Social Handles in the author section 
-Applied DRY rule in the code

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    ></script>
+    <link rel="stylesheet" href="css/styles.css" />
+    <title>Youtube-Clone</title>
+  </head>
+  <body>
+    <nav>
+      <img src="img/logo.jpg" alt="youtube-logo" class="logo" />
+      <div class="search">
+        <input type="search" placeholder="search" class="search-bar" />
+        <img src="img/search.png" alt="magnifying glass" class="magnifier" />
```
Comment (lucasmazo32) [inline index.html]: - [ ] Kindly enclose the image for the magnifying glass into a button tag. Is a requirement for the project.

image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+              <h5>3.9K</h5>
+            </div>
+            <div class="engagement-group dislike-group">
+              <i class="far fa-thumbs-down"></i>
+              <h5>39</h5>
+            </div>
+            <div class="engagement-group share-group">
+              <i class="fas fa-share"></i>
+              <h5>SHARE</h5>
+            </div>
```
Comment (lucasmazo32) [inline index.html]: - [ ] Kindly change the div for a button. It is also a requirement for this project.

image

**Review by lucasmazo32 — CHANGES_REQUESTED** [review]
# Changes required :wrench:

You are doing a great job so far, and the project is looking good. 

Please do not close this PR and use this same link for the next code review; it will help us to keep track on your progress and the changes that need to be done.

For the project to get approved, kindly implement the changes mentioned below.

### Happy coding! :computer:

## Lucas Mazo

**Comment by nusretbutunay** [comment]
- Magnifier is enclosed with a button tag.
- Share button changed from div to a button tag.

**Comment by peterrobert** [comment]
### PULL REQUEST UNDER REVIEW BY PETER ROBERT (TSE)

**Review by peterrobert — APPROVED** [review]
### APPROVED :+1: :100: :1st_place_medal: 
Great work working on the changes that were required. Very impressive readme file :+1:  very impressive pull request and great work following the git-flow :clap: 

Your project is approved to merge and submit the project completion form 

### HAPPY CODING

**Pull request merged**